src/components/Footer/SoilFooterWithContacts.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FC } from 'react'
 import s from '@/components/Footer/SoilFooterWithContacts.module.scss'
 import bitmapPlant from '@/assets/images/bitmap-plant.png'
-import plushLogo from '@/assets/images/footer-logo.png'
+import plushLogo from '@/assets/images/footerLogo/footerLogo@2x.png'
```

Comment:
How logo icon is connected with task "Remove privacy and terms links from footer"? Remove code not related to the task.

Reply:
Explained the situation in Slack. I would have to create almost the same task. The work was carried out on one element.